transports/http/endpoints/api/tips/endpoints.go

```diff
@@ -26,7 +25,7 @@ func (h *handler) RegisterApiEndpoints(router *gin.RouterGroup, cfg *config.HTTP
 	{
 		tip.GET("/tip", h.getTips)
 		tip.GET("/tip/longest", h.getTipLongestChain)
-		tip.GET("/tip/prune/:hash", h.pruneTip)
+		// tip.GET("/tip/prune/:hash", h.pruneTip)
```

Comment:
The same, maybe we could just get rid of it and add JIRA task?

Reply:
removed